lib/pubmed/fetcher.rb

```diff
@@ -31,7 +31,7 @@ def self.fetch_remote_pubmed(pmid)
       # PubMed, oddly enough, the last resort
       pm_xml = Pubmed.client.fetch_records_for_pmid_list(pmid)
       Nokogiri::XML(pm_xml).xpath('//PubmedArticle').map do |doc|
-        add_citation(PubmedSourceRecord.new.source_as_hash(doc))
+        add_citation(Pubmed::MapPubHash.map(doc.to_xml))
       end
```

Comment:
no need to create a new source record in this case just to do a mapping ... since we already have the XML, just call the new class directly